demo/universal/native/lib/ServerFunctions.re

```diff
@@ -66,6 +66,17 @@ module Notes = {
   };
 
   // This is the action generated to be used under the hood for the server and client
+  // Melange don't display the comment after compilation
+  // This is a little hack while we don't have the ppx to generate the comment
+  switch%platform () {
+  | Server => ()
+  | Client => [%mel.raw
+     {|
+    // extract-server-function id/notes/create create Notes
+    ''
+    |}
+    ]
+  };
```

Comment:
While we don't have the ppx, I have to add it with the switch platform, with a little hack adding an empty string cause only `%mel.raw` was working, not the `%%mel.raw`.

In the end, melange will not add the empty string